src/pylogctx/celery.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from celery.app import current_task
+from celery.app.task import Task
+
+from .core import context
+
+
+class LoggingTask(Task):
+    def __call__(self, *args, **kwargs):
+        task = current_task()
+        context.update(
+            celeryTask=task.name,
+            celeryTaskId=task.request.id,
```

Comment:
Okay :) While there are some arguments for stick with camel case (readability, consistency with python code and convenient key names manipulation (it's easy to split by underscore)) I'm going to agree with you.